### ISSUE TITLE
Add some kind of "surcharge" or "handling fee" option

### ISSUE THREAD

**Issue by erommel** [Arsenal21/stripe-payments#103]
Several customers have requested this feature. Use case: user wants to add some kind of "handling and processing" fee (either a fixed fee or some percent of the product price) for the product, but don't want to include it in the product price. One of the examples is here: https://s-plugins.com/forums/topic/default-surcharge/

Tax and\or variations can be used as workaround for this, but in some situations they won't work.

However, if we add percentage support for variations, this might work. At the moment, you can only set fixed amount as price mod for a variation. If we let it use percents as well, this could cover situations like this. E.g. user adds variation named "handling fee" with 5% price mod. The price gets added to the product price on checkout.

**Comment by Arsenal21** [comment]
I think one difference between the variation and this "extra charge" is that the variation is something that a customer selects in terms of a product's option. This "extra charge" is always applied to the transaction/product (regardless).

**Comment by erommel** [comment]
Agree. However, if you only add one variation to the group, there would be nothing to select from ;)

E.g.:
Product fees
Handing and processing - $10

**Comment by Arsenal21** [comment]
True :) 

Then we will have another group of customers who will say, they want variation AND extra charge for the transaction ;)

**Comment by erommel** [comment]
True as well, but we got this covered. Plugin supports multiple variation groups.

E.g.:
Color:
Red
White
Green

Product fees
Handing and processing - $10

**Comment by Arsenal21** [comment]
ah huh! You are right. That should work then.
